src/components/cards/badges/Done.vue

```diff
@@ -0,0 +1,87 @@
+<!--
+  - @copyright Copyright (c) 2022 Thanos Kamber <thanos.kamber@gmail.com>
+  -
+  - @author Thanos Kamber <thanos.kamber@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div v-if="card" class="done">
+		<transition name="zoom">
+			<div v-if="card.done" class="icon-checkmark done-text icon">
```

Comment:
What's the status on these icon classes for nc 25? They don't seem to show for me.

Reply:
cc @juliushaertl

Reply:
Yes, those would need to be migrated to use https://www.npmjs.com/package/vue-material-design-icons

The checkmark icon is already used in https://github.com/nextcloud/deck/blob/0c59a85c9c221ca930044a9831894e663a4cd8af/src/components/cards/CardBadges.vue#L36